stock_picking_tier_validation/models/stock_picking.py

```diff
@@ -0,0 +1,12 @@
+# Copyright Sistemas de Datos
+# Copyright 2020 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class Picking(models.Model):
+    _name = "stock.picking"
+    _inherit = ["stock.picking", "tier.validation"]
+    _state_from = ["draft"]
+    _state_to = ["waiting", "confirmed", "assigned", "done"]
```

Comment:
```suggestion
    _state_from = ["draft", "waiting", "confirmed", "assigned"]
    _state_to = ["done"]
```
@TimLai125 Shouldn't this part be like this?

Reply:
Yes, we should include other states as well since picking created from orders (PO/SO etc.) may already be proceed to assigned state in the beginning. Thanks for the comment.